data/sql/base/db_world/item_template.sql

```diff
@@ -38,25 +38,25 @@ CREATE TABLE `item_template`
   `ContainerSlots` tinyint(3) unsigned NOT NULL DEFAULT '0',
   `StatsCount` tinyint(3) unsigned NOT NULL DEFAULT '0',
   `stat_type1` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  `stat_value1` smallint(6) NOT NULL DEFAULT '0',
+  `stat_value1` int(6) NOT NULL DEFAULT '0',
   `stat_type2` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  `stat_value2` smallint(6) NOT NULL DEFAULT '0',
+  `stat_value2` int(6) NOT NULL DEFAULT '0',
   `stat_type3` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  `stat_value3` smallint(6) NOT NULL DEFAULT '0',
+  `stat_value3` int(6) NOT NULL DEFAULT '0',
   `stat_type4` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  `stat_value4` smallint(6) NOT NULL DEFAULT '0',
+  `stat_value4` int(6) NOT NULL DEFAULT '0',
   `stat_type5` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  `stat_value5` smallint(6) NOT NULL DEFAULT '0',
+  `stat_value5` int(6) NOT NULL DEFAULT '0',
   `stat_type6` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  `stat_value6` smallint(6) NOT NULL DEFAULT '0',
+  `stat_value6` int(6) NOT NULL DEFAULT '0',
   `stat_type7` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  `stat_value7` smallint(6) NOT NULL DEFAULT '0',
+  `stat_value7` int(6) NOT NULL DEFAULT '0',
   `stat_type8` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  `stat_value8` smallint(6) NOT NULL DEFAULT '0',
+  `stat_value8` int(6) NOT NULL DEFAULT '0',
   `stat_type9` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  `stat_value9` smallint(6) NOT NULL DEFAULT '0',
+  `stat_value9` int(6) NOT NULL DEFAULT '0',
   `stat_type10` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  `stat_value10` smallint(6) NOT NULL DEFAULT '0',
+  `stat_value10` int(6) NOT NULL DEFAULT '0',
   `ScalingStatDistribution` smallint(6) NOT NULL DEFAULT '0',
   `ScalingStatValue` int(10) unsigned NOT NULL DEFAULT '0',
   `dmg_min1` float NOT NULL DEFAULT '0',
```

Comment:
you should not edit the base files unless there is a specific need for that (e.g. the file can't be installed)